doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -1349,6 +1349,20 @@ local html = import 'html.libsonnet';
             },
           ],
         },
+        {
+          name: 'flattenDeepArray',
+          params: ['value'],
+          availableSince: 'upcoming',
+          description: |||
+            Concatenate an array containing values and arrays into a single flattened array.
+          |||,
+          examples: [
+            {
+              input: 'std.flattenDeepArray([[1, 2], [], [3, [4]], [[5, 6, [null]], [7, 8]]])',
+              output: std.flattenDeepArray([[1, 2], [], [3, [4]], [[5, 6, [null]], [7, 8]]]),
+            },
+          ],
```

Comment:
I did not understand why the input and output are basically the same but the input has enclosed single quotes. As I didn't have the time to investigate, I'm simply replicating the other examples.

Reply:
Yes it's because we don't have a way to convert an expression to a string. Some languages (e.g. C with its macros) can do that.  But we have to have the expression as a string, and then repeat the expression so it can be computed to a value.

Reply:
Ah, got it. Thank you for the explanation.